src/app/components/LiveLabel/index.tsx

```diff
@@ -0,0 +1,53 @@
+/** @jsx jsx */
+/** @jsxRuntime classic */
+import { PropsWithChildren, useContext } from 'react';
+import { jsx } from '@emotion/react';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import styles from './index.styles';
+
+interface LiveLabelProps {
+  service: string;
```

Comment:
`service` can be deleted from props, as it is no longer required (it was required previously in order to calculate the styling, but we no longer need it)